assets/shiny-server.js

```diff
@@ -388,7 +388,7 @@
       url = url.replace(/\/n=/, '/o=');
     }
     var conn = new SockJS(url,
-      null,{protocols_whitelist: this._whitelist});
+      null,{transports: this._whitelist, sessionId: 16});
```

Comment:
The old session id was 8 characters, yes? Do any REs need changing to allow for more characters?